### PR TITLE
feat: add form-js  extensions blogpost

### LINKED ISSUE #170: Blog about form-js extensions
### What should we do?

We recently added [a bunch of new form-js extension capabilities ](https://github.com/bpmn-io/form-js/issues/123) and made new use cases possible. We should blog about them as they get released via `v1.4.0`.

### Why should we do it?

Provide information about the new form-js extension capabilities.

Cf. https://github.com/bpmn-io/form-js/issues/123
Cf. https://github.com/bpmn-io/form-js-examples/pull/7

### PR DESCRIPTION
**Pull request by pinussilvestrus**
Closes #170

This adds a dedicated blog post about form-js custom components.

Depends on https://github.com/camunda/team-hto/issues/372
Depends on https://github.com/bpmn-io/form-js-examples/pull/7

### CODE REVIEW

**Comment by pinussilvestrus** [comment]
The example is still pending to be merged, but feel free to already review this post 👍

**Comment by barmac** [comment]
Wow, this is so cool! I think this blog post could be published as is.

File: src/blog/2023/005-custom-form-fields.md [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+    Custom properties panel entries for the range component.
+  </p>
+</div>
+
+Find out more about [properties panel extensions](https://github.com/bpmn-io/form-js-examples/tree/master/custom-components#add-custom-properties-panel-entries) in our dedicated example.
+
+
+## Plugging everything together
+
+You can plug your custom modules into the different form-js components as `viewerAdditionalModules` and `editorAdditionalModules`. With the latest release, we also added the `additionalModules` option that allows you to provide extensions to both simultaneously, for example, the renderer.
```
Comment (vsgoulart) [inline src/blog/2023/005-custom-form-fields.md:171]: Should we give an example of `viewerAdditionalModules` too?

Comment (pinussilvestrus) [reply src/blog/2023/005-custom-form-fields.md:171]: I would have to think of a proper use case. In which case you'd like to have an extension for the viewer only, but not for the editor? In terms of a custom renderer you have to add to both anyways. 

One example that comes into my mind is if you want to have a different rendering in both components. We do that for the `text` component, where it looks different in viewer and editor. Maybe that could be something to showcase.

**Review by christian-konrad — APPROVED** [review]
This is so good!

**Comment by nikku** [comment]
Really nice one! :clap:

**Pull request merged**